### PR TITLE
Resolve deploy v2 conflicts

### PR DESCRIPTION
**Pull request by Gidraff**
##### What does this PR do?
This PR resolves merge conflicts on Deploy V2
##### What are the relevant Pivotal Tracker stories?
[Fixes Deploy V2](https://github.com/andela/andela-societies-frontend/pull/222/conflicts)

### CODE REVIEW

**Comment by Kachulio1** [comment]
@Gidraff there are 489 files change

**Comment by Kachulio1** [comment]
Please raise a PR with only your changes and make sure you don't format the whole codebase with Prettier 😉

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-[![CircleCI](https://circleci.com/gh/andela/andela-societies-frontend.svg?style=svg)](https://circleci.com/gh/andela/andela-societies-frontend)
-[![Coverage Status](https://coveralls.io/repos/github/andela/andela-societies-frontend/badge.svg)](https://coveralls.io/github/andela/andela-societies-frontend)
+[![CircleCI](https://circleci.com/gh/andela/andela-societies-frontend/tree/develop-V2.svg?style=svg)](https://circleci.com/gh/andela/andela-societies-frontend/tree/develop-V2)
```
Comment (WinstonKamau) [inline README.md]: @Kachulio1 are we deprecating develop-v2 after the merge, and using develop solely?

Comment (Kachulio1) [reply README.md]: Yes ...

Comment (Kachulio1) [reply README.md]: 😕

Comment (WinstonKamau) [reply README.md]: @Gidraff I'd suggest we leave the CircleCI badges and Coverage badges as before and remove the develop-v2 changes.

**Comment by alexmochu** [comment]
> Please raise a PR with only your changes and make sure you don't format the whole codebase with Prettier 😉
 
+1 on this.

File: README.md (hunk 1 above)
Comment (Gidraff) [reply README.md]: @WinstonKamau Okay. Working on it.

**Pull request merged**